src/common/classes/fb_string.h

```diff
@@ -411,10 +411,10 @@ namespace Firebird
 		}
 		void insert(size_type p0, const_pointer s, const size_type n)
 		{
```

Comment:
On 06/16/2016 07:58 PM, Dimitry Sibiryakov wrote:
>> @@ -411,10 +411,10 @@ namespace Firebird
>>   		}
>>   		void insert(size_type p0, const_pointer s, const size_type n)
>>   		{
> No, this is regression of this branch, problem does not exist in other branches.

OK, in that case no problems.